services/api-server/tests/unit/_with_db/conftest.py

```diff
@@ -269,5 +269,5 @@ async def auth(
 ) -> httpx.BasicAuth:
     """overrides auth and uses access to real repositories instead of mocks"""
     async for key in create_fake_api_keys(1):
-        return httpx.BasicAuth(key.api_key, key.api_secret.get_secret_value())
-    assert False, "Did not generate authentication"
+        return httpx.BasicAuth(key.api_key, key.api_secret)
```

Comment:
Are you sure you do not need `.get_secret_value()`?? Please double check.

Reply:
Triple checked this. To me it looks good as is currently.